spec/integration/authorization/pundit_spec.rb

```diff
@@ -0,0 +1,163 @@
+require 'spec_helper'
+include Pundit
+
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def rails_admin?(action)
```

Comment:
How about [this issue](https://github.com/sferik/rails_admin/pull/2399#discussion_r39257553)?
We really need to get rid of this ugly case-when chain.